pkg/server/service.go

```diff
@@ -45,6 +45,11 @@ import (
 	snapshotstore "github.com/containerd/cri/pkg/store/snapshot"
 )
 
+const (
+	// defaultIfName is the default network interface for the pods
+	defaultIfName = "eth0"
```

Comment:
Put this into helpers.go? Now all constants like this are in helpers.

Reply:
will do